api/src/test/java/org/apache/iceberg/AssertHelpers.java

```diff
@@ -28,7 +30,8 @@ private AssertHelpers() {
   }
 
   /**
-   * A convenience method to avoid a large number of @Test(expected=...) tests
+   * A convenience method to avoid a large numb/test/java/org/apache/iceberg/TestHelpers.java
```

Comment:
This comment looks like the victim of an unintended search and replace.

Reply:
Good catch, thanks!